Makefile

```diff
@@ -1,6 +1,6 @@
 TESTIMONY_TOKENS="caseautomation, casecomponent, caseimportance, caselevel, caseposneg, description, expectedresults, id, requirement, setup, subtype1, subtype2, steps, teardown, testtype, upstream, title"
 TESTIMONY_MINIMUM_TOKENS="id, requirement, caseautomation, caselevel, casecomponent, testtype, caseimportance, upstream"
-PYTEST_OPTIONS=--verbose
+PYTEST_OPTIONS=--verbose --driver=Remote
```

Comment:
I meant that since pytest-selenium is not being used anymnore pytest without pytest-selenium installed will fail:

```
$ pytest --driver Chrome
usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --driver
  inifile: None
  rootdir: /home/elyezer/code/shiftigrade
```

You may have needed to add this because the ui/conftest.py has the following hook:

```python
def pytest_collection_modifyitems(config, items):
    """Check for presence of driver for selenium.

    If no driver option is present then UI test should be skipped.
    UI tests should be marked with @pytest.mark.ui
    """
    if not config.getoption('--driver', None):
        skip_ui = pytest.mark.skip(
            reason='need --driver option to run UI tests')
        for item in items:
            if 'ui' in item.fspath.strpath:
                item.add_marker(skip_ui)
```

Which skips all UI if the --driver is not defined. Since you are relying on environment variables to select the driver you need to adjust the hook to look for the env vars instead of the `--driver` pytest option. Having the driver always there will run UI tests even when that it not intended.